content/cli/commands/cluster.md

```diff
@@ -1,5 +1,266 @@
 ---
-title: Fluvio Runner
-menu: Runner
-weight: 10
+title: Fluvio Cluster
+menu: Cluster
+weight: 50
 ---
+
+The `fluvio cluster` family of commands is used to install and manage your own
+Fluvio cluster. The two primary forms of self-hosted cluster are Kubernetes
+(via Minikube) and Local clusters, both of which can be automatically set up
+by cluster commands. Please make sure you have followed the
+[getting started guide] and installed all the necessary dependencies before
+attempting to install your own cluster.
+
+[getting started guide]: /docs/get-started
+
+## `fluvio cluster check`
+
+This command is used to check whether you have all the required dependencies
+set up on your system and whether they are running correctly.
+If this runs and returns successfully, you should be all set to start
+a local or minikube cluster.
+
+```
+fluvio-cluster-check
+Check that all requirements for cluster startup are met
+
+USAGE:
+    fluvio cluster check
+
+FLAGS:
+    -h, --help    Prints help information
+```
+
+Example usage:
+
+```bash
+$ fluvio cluster check
+Running pre-startup checks...
+✅ ok: Kubernetes config is loadable
+✅ ok: Supported kubernetes version is installed
+✅ ok: Supported helm version is installed
+✅ ok: Fluvio system charts are installed
+✅ ok: Can create service
+✅ ok: Can create customresourcedefinitions
+✅ ok: Can create secret
+✅ ok: Load balancer is up
+All checks passed!
+You may proceed with cluster startup
+next: run `fluvio cluster start`
+```
+
+## `fluvio cluster start`
+
+This command is used to start your own Fluvio cluster, with all the
+machinery needed to receive, process, and serve streaming messages.
+
+There are two main variations of this command. The default variation
```

Comment:
add bullet points for easier readability

- `fluvio cluster start` ...
- `fluvio cluster start --local` ...